### PR TITLE
8318736: com/sun/jdi/JdwpOnThrowTest.java failed with "transport error 202: bind failed: Address already in use"

### PR DESCRIPTION
**Pull request by parttimenerd**
Tested it locally.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] [JDK-8318736](https://bugs.openjdk.org/browse/JDK-8318736) needs maintainer approval

### Issue
 * [JDK-8318736](https://bugs.openjdk.org/browse/JDK-8318736): com/sun/jdi/JdwpOnThrowTest.java failed with "transport error 202: bind failed: Address already in use" (**Bug** - P4 - Approved)

### Reviewers
 * [Matthias Baesken](https://openjdk.org/census#mbaesken) (@MBaesken - **Reviewer**) ⚠️ Review applies to [391328e1](https://git.openjdk.org/jdk11u-dev/pull/2453/files/391328e18356cbcb310fb771fc0abf52a4f73ed0)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk11u-dev.git pull/2453/head:pull/2453` \
`$ git checkout pull/2453`

Update a local copy of the PR: \
`$ git checkout pull/2453` \
`$ git pull https://git.openjdk.org/jdk11u-dev.git pull/2453/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 2453`

View PR using the GUI difftool: \
`$ git pr show -t 2453`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk11u-dev/pull/2453.diff">https://git.openjdk.org/jdk11u-dev/pull/2453.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk11u-dev/pull/2453#issuecomment-1885165921)

### CODE REVIEW

**Review by MBaesken — COMMENTED** [review]
what happened to the file ending of [test/jdk/com/sun/jdi/JdwpOnThrowTest.java](https://github.com/openjdk/jdk11u-dev/pull/2453/files#diff-717b78b152e4f13ecdf354c4f896b7dcf6f6615c12bb06683018b38cca0487d3)  ?
I do not see a  "minus" at the end in 17u-dev version of the change.

**Comment by parttimenerd** [comment]
I don't know. I added newlines.

**Comment by MBaesken** [comment]
> I don't know. I added newlines.

I think you only need the line break , no additional complete empty line. At least I do not see this in jdk-head .

**Comment by MBaesken** [comment]
Wondering why we see the macOS GHA errors.
Is 
https://github.com/openjdk/jdk11u-dev/pull/2464
not yet in ? This should fix it for jdk11u-dev .

**Comment by parttimenerd** [comment]
How do I add a line-break that isn't a new line? I'm confused.

**Comment by parttimenerd** [comment]
Fixed the new lines.

File: test/jdk/com/sun/jdi/lib/jdb/Debuggee.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, 2024, Oracle and/or its affiliates. All rights reserved.
```
Comment (MBaesken) [inline test/jdk/com/sun/jdi/lib/jdb/Debuggee.java:2]: the jdk17u-dev change  (https://github.com/openjdk/jdk17u-dev/pull/1949/files)  has  Copyright (c) 2018, 2023,
so why 2024 at this place?

**Review by MBaesken — APPROVED** [review]
looks okay, still wondering a bit about the COPYRIGHT 2024 in Debugee.java, this is newer than 17u-dev ?

File: test/jdk/com/sun/jdi/lib/jdb/Debuggee.java [hunk 2]
```diff
@@ -211,5 +233,4 @@ public void close() throws IOException {
             p.destroy();
         }
     }
-
-}
+}
```
[the remark this answers is not in the document]
Comment (parttimenerd) [reply test/jdk/com/sun/jdi/lib/jdb/Debuggee.java]: No file ending changes with this patch anymore :)

**Comment by parttimenerd** [comment]
/approval request Fixes a bug in the JDWP tests.
Tested in the SapMachine nightly tests and on my local machine. Low risk.

**Pull request closed without merging**